cmd/goal/commands.go

```diff
@@ -220,6 +221,16 @@ var reportCmd = &cobra.Command{
 	},
 }
 
+var protoCmd = &cobra.Command{
+	Use:   "protocols",
+	Short: "",
+	Long:  "Dump known consensus protocols as json to stdout.",
```

Comment:
could you rephrase it to "Dump built-in consensus.." ?
We do support dynamically loaded consensus protocols, so saying "built-in" would better describe these.

Reply:
how about "standard",  "Dump standard consensus protocols ..."

Reply:
Sounds good